Correction of mistakes in Russian translation

CCCP84 opened the pull request:
I uploaded the file to GPT chat and it found some mistakes.
Here are the corrections.

CCCP84 committed:
Correction of mistakes in Russian translation

CCCP84 committed:
Add author's name

CCCP84 commented:
@MahBoiDeveloper Are you really the author of this translation?

MahBoiDeveloper commented on package/Resources/Translations/ru/Translation.ini:
```diff
@@ -53,7 +53,7 @@ Client:DTAConfig:DetailLevelLow=Низкая; Low
 Client:DTAConfig:DisablePMPopup=Отключить всплывающее окно@Личных Сообщений; Disable Popups from Private Messages
 Client:DTAConfig:DiscordStatus=Показывать детальную информацию@об игре в статусе Discord; Show detailed game info in Discord status
-Client:DTAConfig:DownloadCompleteText=Скачивание дополнительного компонента {0} успешно завершено.;Download of optional component {0} completed succesfully.
+Client:DTAConfig:DownloadCompleteText=Скачивание дополнительного компонента {0} успешно завершено.;Download of optional component {0} completed successfully.
```
This typo in the comment doesn't affect to value of the ini key

CCCP84 replied:
GPT chat fixed it automatically

MahBoiDeveloper commented on package/Resources/Translations/ru/Translation.ini:
```diff
@@ -89,10 +89,10 @@ Client:DTAConfig:PMFriends=Друзья; Friends
 Client:DTAConfig:PressAKey=Нажмите сочетание клавиш...;Press a key...
 Client:DTAConfig:Recommended=(рекомендуется);(recommended)
-Client:DTAConfig:ReconnectAfterRename=* Если вы сейчас соединены с CnCNet, то вам необходимо разлогинится@и переподключиться, чтобы ваш новый никнейм был использован.; * If you are currently connected to CnCNet, you need to log out and reconnect@for your new name to be applied.
+Client:DTAConfig:ReconnectAfterRename=* Если вы сейчас соединены с CnCNet, то вам необходимо отключиться от сети@и переподключиться, чтобы ваш новый никнейм был использован.; * If you are currently connected to CnCNet, you need to log out and reconnect@for your new name to be applied.
```
Rephrasizing is less correct than original text. You need to click log out button to change the name.

CCCP84 replied:
+++

MahBoiDeveloper commented on package/Resources/Translations/ru/Translation.ini:
```diff
@@ -89,10 +89,10 @@ Client:DTAConfig:PMFriends=Друзья; Friends
 Client:DTAConfig:Renderer=Враппер/Отрисовщик:;Renderer:
 Client:DTAConfig:ResetAllHotkey=Сбросить всё;Reset All Keys
-Client:DTAConfig:ResetToDefault=Сбосить;Reset to Default
+Client:DTAConfig:ResetToDefault=Сбросить;Reset to Default
```
Good catch

CCCP84 replied:
+++

MahBoiDeveloper commented on package/Resources/Translations/ru/Translation.ini:
```diff
@@ -136,7 +136,7 @@ Client:DTAConfig:TSFixInstallFailText=Ошибка установки испра
 Client:DTAConfig:TSFixInstallSuccessText=Исправление совместимости DTA/TI/TS было успешно установлено.;The DTA/TI/TS Compatibility Fix has been succesfully installed.
 Client:DTAConfig:TSFixInstallSuccessTitle=Установлено исправление совместимости;Compatibility Fix Installed
-Client:DTAConfig:TSFixText=Вы всегда сможете установить и удалить последние исправление совместимости из меню настроек.;You'll always be able to install or uninstall the compatibility fix later from the options menu.
+Client:DTAConfig:TSFixText=Вы всегда сможете установить и удалить последние исправление совместимости через меню настроек.;You'll always be able to install or uninstall the compatibility fix later from the options menu.
```
Good one

CCCP84 replied:
+++

MahBoiDeveloper commented:
> @MahBoiDeveloper Are you really the author of this translation?

You can see file changes by authors in git blame.

MahBoiDeveloper commented on package/Resources/Translations/ru/Translation.ini:
```diff
@@ -170,7 +170,7 @@ Client:Main:AttemptJoin=Попытка присоединения к лобби
 Client:Main:AuthorByUnknown=Автор: Неизвестно; By Unknown Author
 Client:Main:AutoAllyingPrefix=Автоматические союзы:;Auto Allying:
-Client:Main:AutoAllyingText1=Автоматические союзы позволяют хосту привязать стартовые позиции к командам, а не к игрокам.@Когда игроки спавнятся на привязанной позиции, они автоматически присоединяются к соответствующей команде.@Это лучший способ рандомизации команд и позиций. однако для это требуется только случайные команды.@Позиции, заданные вручную, будут иметь приоритет.@@;Auto allying allow ...
+Client:Main:AutoAllyingText1=Автоматические союзы позволяют хосту привязать стартовые позиции к командам, а не к игрокам.@Когда игроки спавнятся на привязанной позиции, они автоматически присоединяются к соответствующей команде.@Это лучший способ рандомизации команд и позиций. однако для этого требуются только случайные команды.@Позиции, заданные вручную, будут иметь приоритет.@@;Auto allying all ...
```
Fix `. однако` to `. Однако` please

CCCP84 replied:
+++

MahBoiDeveloper commented on package/Resources/Translations/ru/Translation.ini:
```diff
@@ -432,11 +432,11 @@ Client:Main:InvalidPlayerName=Неправильный никнейм;Invalid Pl
 Client:Main:InviteNotDelivered={0} не может получить ваше приглашение. Игрок уже в лобби, либо принимает приглашения только от друзей. Убедитесь, что ваше лобби не закрыто и доступно из лобби CnCNet перед тем, как делать очередную попытку.;{0} could not receive your invitation. They might be in game or only accepting invitations from friends. Ensure your game is unlocked and visible in the lobby  ...
 Client:Main:Join=Присоединиться;Join
-Client:Main:JoinFailedText=Невозможно присоединиться к лобби хоста {0}. Лобби закрыто или удалено.;Unable to join {0}'s game. The game may be locked or closed.
+Client:Main:JoinFailedText=Невозможно присоединиться к лобби игрока {0}. Лобби закрыто или удалено.;Unable to join {0}'s game. The game may be locked or closed.
```
Rephrasing is less correct

CCCP84 replied:
"Host lobby" in Russian will not be clear to everyone.
I leave as is "player lobby"

MahBoiDeveloper replied:
> "Host lobby" in Russian will not be clear to everyone.
> I leave as is "player lobby"

It is a clear that host (lobby admin) opens the game room. In Russian we have verbs "хостить", "хостится", "схостить", "схостил" commonly used by the gamers community.

MahBoiDeveloper commented on package/Resources/Translations/ru/Translation.ini:
```diff
@@ -432,11 +432,11 @@ Client:Main:InvalidPlayerName=Неправильный никнейм;Invalid Pl
 Client:Main:JoinGame=Присоединиться;Join Game
 Client:Main:JoinGameErrorGameRunning=Невозможно присоединиться к лобби, пока игра запущена.;Cannot join game while the main game executable is running.
-Client:Main:JoinGameErrorInProgress=Невозможно присоединиться к лобби - идёт присоединение. Если вы считаете, что это ошибка, пожалуйста, разлогиньтесь и войдите в сеть обратно.;Cannot join game - joining game in progress. If you believe this is an error, please log out and back in.
+Client:Main:JoinGameErrorInProgress=Невозможно присоединиться к лобби - уже идёт присоединение. Если вы считаете, что это ошибка, пожалуйста, разлогиньтесь и войдите в сеть обратно.;Cannot join game - joining game in progress. If you believe this is an error, please log out and back in.
```
`Невозможно присоединиться к лобби, т.к. уже идёт попытка соединения`

CCCP84 replied:
+++

MahBoiDeveloper reviewed (changes requested):
Add changes from my notes, please.

CCCP84 committed:
Corrections

CCCP84 commented:
> > @MahBoiDeveloper Are you really the author of this translation?
> 
> You can see file changes by authors in git blame.

I don't mean the author of the file change, but the author of the Russian translation ;)

MahBoiDeveloper commented:
> > > @MahBoiDeveloper Are you really the author of this translation?
> > 
> > You can see file changes by authors in git blame.
> 
> I don't mean the author of the file change, but the author of the Russian translation ;)

Yes

MahBoiDeveloper commented on package/Resources/Translations/ru/Translation.ini:
```diff
@@ -89,10 +89,10 @@ Client:DTAConfig:PMFriends=Друзья; Friends
 Client:DTAConfig:PressAKey=Нажмите сочетание клавиш...;Press a key...
 Client:DTAConfig:Recommended=(рекомендуется);(recommended)
-Client:DTAConfig:ReconnectAfterRename=* Если вы сейчас соединены с CnCNet, то вам необходимо разлогинится@и переподключиться, чтобы ваш новый никнейм был использован.; * If you are currently connected to CnCNet, you need to log out and reconnect@for your new name to be applied.
+Client:DTAConfig:ReconnectAfterRename=* Если вы сейчас соединены с CnCNet, то вам необходимо выйти из аккаунта@и переподключиться, чтобы ваш новый никнейм был использован.; * If you are currently connected to CnCNet, you need to log out and reconnect@for your new name to be applied.
```
`Разлогиниться` is also a button text, when user have enabled "Stay connected" option. I suggest to return back to previous version.

MahBoiDeveloper replied:
@CCCP84 please leave a comment

CCCP84 replied:
returned back

CnCRAZER commented:
Do you speak Russian? lol

Edit: @MahBoiDeveloper do you speak russian?

MahBoiDeveloper commented:
> Do you speak Russian? lol
> 
> Edit: @MahBoiDeveloper do you speak russian? 

Yes

MahBoiDeveloper commented:
@CCCP84 Do you have any progress with PR?

CCCP84 commented:
> @CCCP84 Do you have any progress with PR?

What progress? All is done.

MahBoiDeveloper commented:
> > @CCCP84 Do you have any progress with PR?
> 
> What progress? All is done.

Check review comments.

CCCP84 committed:
Update Translation.ini

Fix lobby open/close (was reversed)

CCCP84 commented:
I can't request reviews from others. There is no button.

MahBoiDeveloper commented on package/Resources/Translations/ru/Translation.ini:
```diff
@@ -432,11 +432,11 @@ Client:Main:InvalidPlayerName=Неправильный никнейм;Invalid Pl
 Client:Main:JoinGame=Присоединиться;Join Game
 Client:Main:JoinGameErrorGameRunning=Невозможно присоединиться к лобби, пока игра запущена.;Cannot join game while the main game executable is running.
-Client:Main:JoinGameErrorInProgress=Невозможно присоединиться к лобби - идёт присоединение. Если вы считаете, что это ошибка, пожалуйста, разлогиньтесь и войдите в сеть обратно.;Cannot join game - joining game in progress. If you believe this is an error, please log out and back in.
+Client:Main:JoinGameErrorInProgress=Невозможно присоединиться к лобби - уже идёт попытка присоединения. Если вы считаете, что это ошибка, пожалуйста, разлогиньтесь и войдите в сеть обратно.;Cannot join game - joining game attempt in progress. If you believe this is an error, please log out and back in.
```
`Невозможно присоединиться к лобби - уже идёт попытка соединения` sounds better.

CCCP84 replied:
changed

CCCP84 replied:
but i am not sure this is right..

CCCP84 replied:
connection with lobby?

MahBoiDeveloper replied:
> connection with lobby?

yes, connecting with lobby

CCCP84 committed:
Update Translation.ini

CCCP84 committed:
Update Translation.ini

CCCP84 committed:
Update Translation.ini

CCCP84 commented:
I think, it is over. It can be merged and closed

CCCP84 commented:
@MahBoiDeveloper Don’t you think the ‘Network Play’ button should be in place of LAN? And instead of it, there should be ‘Internet Play’?

CCCP84 commented:
Or just "internet", "network" buttons
Or "Play online", "Network"
Or ""Play online", "Play via Network"

CCCP84 commented:
@MahBoiDeveloper In main lobby, in text color selection drop-down purple and pink are not translated. 
Where should it be changed? I can't find

CCCP84 committed:
Update Translation.ini

MahBoiDeveloper commented:
> Or just "internet", "network" buttons Or "Play online", "Network" Or ""Play online", "Play via Network"

Original `Play Online` should be stay as is. If you want to make change in wording playing online through CnCNet, make edits into English version first

CCCP84 commented:
Fixed a typo in the word ‘orange’, and removed the ‘Medium’ prefix for purple and the ‘Deep’ prefix for pink. It’s an improvisation, but it might work, who knows... Let’s see.

CCCP84 commented:
> > Or just "internet", "network" buttons Or "Play online", "Network" Or ""Play online", "Play via Network"
> 
> Original `Play Online` should be stay as is. If you want to make change in wording playing online through CnCNet, make edits into English version first

You made translation not "play online". Now there is "Play via network" (Играть по сети)

MahBoiDeveloper commented:
> @MahBoiDeveloper In main lobby, in text color selection drop-down purple and pink are not translated. Where should it be changed? I can't find

I think new colors comes with one of the client binaries update.

CCCP84 commented:
I think, it can be merged. If we will find some other problems, we will make other PR.

CCCP84 commented:
This pull request should be merged as soon as possible because there are critical translation errors in the client now. When the game room opens, there is a message that the room is closed and vice versa. Players do not understand what is happening. My friends have already told me about this.
@brichardson1991
And why can't I request a review? Do I not have enough rights? Can someone enable the ability to request a review?

brichardson1991 commented:
> This pull request should be merged as soon as possible because there are critical translation errors in the client now. When the game room opens, there is a message that the room is closed and vice versa. Players do not understand what is happening. My friends have already told me about this. @brichardson1991 

You understand, that I don't have any knowledge of Russian language so I have to wait for others to review, I can't just blindly pull language files in. By the looks of it, there's been a lot of changes, so again waiting for someone to confirm it's completed.

> And why can't I request a review? Do I not have enough rights? Can someone enable the ability to request a review?

Only people inside the CnCNet GitHub organization can be added to request reviews but that doesn't mean you can't review the changes manually and comment here.

brichardson1991 commented:
@MahBoiDeveloper  is this good to ship?

MahBoiDeveloper commented:
> @MahBoiDeveloper  is this good to ship?

Yes, feel free to merge.
